data/sources/admin-boundaries.json

```diff
@@ -22,6 +22,11 @@
       "sql": "SELECT the_geom_webmercator, boroname FROM dcp_borough_boundary",
       "dataPipeline": false
     },
+    {
+      "id": "zip-codes",
+      "sql": "SELECT the_geom_webmercator, zipcode FROM nyc_zip_codes",
+      "dataPipeline": true
```

Comment:
Only nit is to change these  `dataPipeline` values to `false`.  A `true` value would mean it is maintained by GIS/EDM. But I believe these layers `nyc_zip_codes` and `nycdta2020` are one-off shapefiles we just recently uploaded to carto?